__tests__/index.test.js

```diff
@@ -15,3 +17,16 @@ test("Renders the correct label for my input in the form component", () => {
   const element = screen.getByLabelText("Search a movie:");
   expect(element).toBeInTheDocument();
 });
+
+const movie = {
+  id: 1,
+  title: "Movie-Title",
+  release_date: "2023-03-21",
+  genre_ids: [12, 18],
+};
+
+test("Renders the Movie component with the details", () => {
+  render(<Movie key={movie.id} movie={movie} />);
+  const element = screen.getByRole("heading", { name: /movie-title/i });
+  expect(element).toBeInTheDocument();
+});
```

Comment:
Maybe you could test a bit more, than just, is the elememtn in the document. Because this tests just, there is something. Could be an empty `div`. Highly recommend testing here if the Heading really contains the "Movie-Title", and are the genres correct? (12 === Adventure and 18 === Drama ).

Could also test, what if genre nummer not found? is the component behaving correctly then?